content.js

```
@@ -0,0 +1,262 @@
+            if (button.parentNode) {
+                button.remove();
+            }
+        }, 10000);
+    }
+    
+    // Listener para mensagens do background script
+    chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+        switch (request.action) {
+            case 'insertPrompt':
+                const success = insertTextInField(request.text);
+                sendResponse({ success });
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)